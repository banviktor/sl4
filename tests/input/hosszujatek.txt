randomization off
questions off
start_game 2
8 3
5 6
inputvector -5 0
jump
jump
N
inputvector 6 0
jump
jump
N
inputvector 6 0
jump
jump
N
inputvector -5 0
jump
jump
N
inputvector -5 0
jump
jump
N
inputvector 6 0
jump
jump
N
inputvector 6 0
jump
jump
N
inputvector -5 0
jump
jump
N
inputvector -5 0
jump
jump
N
inputvector 6 0
jump
jump
N
inputvector 6 0
jump
jump
N
inputvector -5 0
jump
jump
N
inputvector -5 0
jump
jump
N
inputvector 6 0
jump
jump
N
inputvector 6 0
jump
jump
N
inputvector -5 0
jump
jump
N
inputvector -5 0
jump
jump
N
inputvector 6 0
jump
jump
N
inputvector 6 0
jump
jump
N
inputvector -5 0
jump
jump
exit
