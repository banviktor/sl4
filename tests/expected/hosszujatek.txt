Véletlenszerű események kikapcsolva.
Kérdezés kikapcsolva.
Pálya betöltve.
Új játék elkezdve 2 játékossal.
Ez a(z) 1. kör.
A Piros Robot következik.
Inputvector: -1.0, 0.0
Új pozíció: 7.0, 3.0
Ez a(z) 1. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 2. kör.
A Piros Robot következik.
Inputvector: 1.0, 0.0
Új pozíció: 7.0, 3.0
Ez a(z) 2. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 3. kör.
A Piros Robot következik.
Inputvector: 1.0, 0.0
Új pozíció: 8.0, 3.0
Ez a(z) 3. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 4. kör.
A Piros Robot következik.
Inputvector: -1.0, 0.0
Új pozíció: 8.0, 3.0
Ez a(z) 4. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 5. kör.
A Piros Robot következik.
Inputvector: -1.0, 0.0
Új pozíció: 7.0, 3.0
Ez a(z) 5. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 6. kör.
A Piros Robot következik.
Inputvector: 1.0, 0.0
Új pozíció: 7.0, 3.0
Ez a(z) 6. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 7. kör.
A Piros Robot következik.
Inputvector: 1.0, 0.0
Új pozíció: 8.0, 3.0
Ez a(z) 7. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 8. kör.
A Piros Robot következik.
Inputvector: -1.0, 0.0
Új pozíció: 8.0, 3.0
Ez a(z) 8. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 9. kör.
A Piros Robot következik.
Inputvector: -1.0, 0.0
Új pozíció: 7.0, 3.0
Ez a(z) 9. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 10. kör.
A Piros Robot következik.
Inputvector: 1.0, 0.0
Új pozíció: 7.0, 3.0
Ez a(z) 10. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 11. kör.
A Piros Robot következik.
Inputvector: 1.0, 0.0
Új pozíció: 8.0, 3.0
Ez a(z) 11. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 12. kör.
A Piros Robot következik.
Inputvector: -1.0, 0.0
Új pozíció: 8.0, 3.0
Ez a(z) 12. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 13. kör.
A Piros Robot következik.
Inputvector: -1.0, 0.0
Új pozíció: 7.0, 3.0
Ez a(z) 13. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 14. kör.
A Piros Robot következik.
Inputvector: 1.0, 0.0
Új pozíció: 7.0, 3.0
Ez a(z) 14. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 15. kör.
A Piros Robot következik.
Inputvector: 1.0, 0.0
Új pozíció: 8.0, 3.0
Ez a(z) 15. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 16. kör.
A Piros Robot következik.
Inputvector: -1.0, 0.0
Új pozíció: 8.0, 3.0
Ez a(z) 16. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 17. kör.
A Piros Robot következik.
Inputvector: -1.0, 0.0
Új pozíció: 7.0, 3.0
Ez a(z) 17. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 18. kör.
A Piros Robot következik.
Inputvector: 1.0, 0.0
Új pozíció: 7.0, 3.0
Ez a(z) 18. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 19. kör.
A Piros Robot következik.
Inputvector: 1.0, 0.0
Új pozíció: 8.0, 3.0
Ez a(z) 19. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
Ez a(z) 20. kör.
A Piros Robot következik.
Inputvector: -1.0, 0.0
Új pozíció: 8.0, 3.0
Ez a(z) 20. kör.
A Sárga Robot következik.
Új pozíció: 5.0, 6.0
A Piros Robot nyert!
